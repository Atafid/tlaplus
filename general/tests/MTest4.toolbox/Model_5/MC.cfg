SPECIFICATION spec_124951403175825000
\* Generated at Wed Aug 05 16:13:51 PDT 2009