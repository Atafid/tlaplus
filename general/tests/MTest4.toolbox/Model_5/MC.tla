---- MODULE MC ----
EXTENDS MTest4, TLC

\* Specification formula
spec_124951403175825000 ==
(x=0) /\[][ FALSE ]_<<x, y>> 
----
====
\* Generated at Wed Aug 05 16:13:51 PDT 2009