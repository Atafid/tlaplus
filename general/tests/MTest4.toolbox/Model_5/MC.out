TLC2 Version 2.02 of 3 August 2009
@!@!@STARTMSG 2187:0 @!@!@
Running in Model-Checking mode.
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file MC.tla
Parsing file MTest4.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\TLC.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Naturals.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Sequences.tla
Semantic processing of module Naturals
Semantic processing of module MTest4
Semantic processing of module Sequences
Semantic processing of module TLC
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting...
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2102:1 @!@!@
/\ x = 0
/\ y = null

While working on the initial state:
%2%
@!@!@ENDMSG 2102 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished.
@!@!@ENDMSG 2186 @!@!@
